nucypher/blockchain/eth/interfaces.py

```diff
@@ -401,18 +427,38 @@ def sign_and_broadcast_transaction(self,
                                        ) -> dict:
 
         #
-        # Sign
+        # Setup
         #
 
+        # TODO # 1754
+        # TODO: Move this to singleton - I do not approve... nor does Bogdan?
+        if GlobalLoggerSettings._json_ipc:
+            emitter = JSONRPCStdoutEmitter()
+        else:
+            emitter = StdoutEmitter()
```

Comment:
I'd rather prefer to have some sort of singleton global context to retrieve an already instantiated emitter (if any), like when we are using a CLI. Anyway, since #1754 is open, this comment is out of the scope of this PR.